include/crypto/ecx.h

```diff
@@ -109,10 +109,19 @@ int X448(uint8_t out_shared_key[56], const uint8_t private_key[56],
 void X448_public_from_private(uint8_t out_public_value[56],
                               const uint8_t private_key[56]);
 
+
 /* Backend support */
 int ecx_public_from_private(ECX_KEY *key);
 int ecx_key_fromdata(ECX_KEY *ecx, const OSSL_PARAM params[],
                      int include_private);
 
+ECX_KEY *evp_pkey_get0_X25519(const EVP_PKEY *pkey);
```

Comment:
This either needs to go in crypto/evp.h or it needs to be renamed to
ecx_key_

Reply:
I disagree...  they return ECX_KEY pointers, so they can be claimed to fit here just as well.
Furthermore, `crypto/evp.h` contains a little too much stuff that can lead to undesired shortcuts, and the decoders already include `crypto/ecx.h`.  Also, if you look in files like `include/openssl/ec.h`, you'll find EVP functions / macros there.
Also, considering this is internal anyway, I don't see a need to be overly fussier...